README.md

```
@@ -43,6 +40,10 @@
  - [**Team Members**](#-team-members)
  - [**Product Tour**](#-product-tour)
  - [**Features**](#-features)
+ - [**🛠 Project Design Overview**](#-project-design-overview)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)